cloudscheduler/csmain

```diff
@@ -1240,7 +1240,6 @@ def remove_orphaned_volumes(config):
                             rc, msg, clouds = config.db_query(CLOUD, where=where_clause)
                             cloud=clouds[0]
                             region = cloud["region"] if cloud.get("region") else None
-                            verify = cloud["cacertificate"] if cloud.get("cacertificate") else None
                             sess = get_openstack_sess(cloud, verify)
```

Comment:
verify needs to be removed from the openstack session request since it's now no longer defined. It's an optional parameter so you should be able to request the session with just "cloud"